game/world/worldobjects.cpp

```diff
@@ -161,9 +161,15 @@ void WorldObjects::tick(uint64_t dt, uint64_t dtPlayer) {
 
   for(size_t i=0; i<npcArr.size(); ++i) {
     auto& npc = *npcArr[i];
-    if(npc.isPlayer())
-      npc.tick(dtPlayer); else
+    if(npc.isPlayer()) {
+      if(Gothic::inst().camera() && Gothic::inst().camera()->isMarvinMode(Camera::M_Free)) {
+        return;
```

Comment:
Also why player is not receiving ticks, when in free-cam mode?

Reply:
Player can't move, doesn't drop if in air, receives no damage if diving bar is empty, `aigoto` only starts if `free cam` mode is left and current animation is frozen. Player can be damaged by monsters and npc though but death animation behaves like `aigoto`. Maybe it's best to just activate `godmode` here. 

So basically all what's done in player-tick is skipped/ignored.

Reply:
Tested vanilla today:
In free cam move playable npc doesn't receive animation ticks for skeletal animation, an no physics ticks. It can receive damage and perceptions(magic damage at least).
Also main menu not working in free-camera O-o

Anyway `return;` is definitely wrong here.